models/migrations/v205.go

```diff
@@ -0,0 +1,18 @@
+// Copyright 2021 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package migrations
+
+import (
+	"xorm.io/xorm"
+)
+
+func addAllowEditsFromMaintainers(x *xorm.Engine) error {
+	// PullRequest represents relation between pull request and repositories.
+	type PullRequest struct {
+		AllowEditsFromMaintainers bool
```

Comment:
It should be `not null` with `default true`

Reply:
I don't think `default true` is a good idea. This will make thousands of pull request branches accessible for people, even if they don't want that it's enabled. `not null` is understandable, but it's actually useless if you use `false` as default.

Reply:
It's not useless as even with default false it would be still possible to end up with null value in column and that could break something in future

Reply:
Default can be discussed but imho githubs default behavior is also to allow edit for maintainers. So if you have created PR I don't see why you would not want to allow edit for maintaincers at least most of the time, at the moment I can't think at the top of my head a reason why you would disallow it.